carbonmark/components/pages/Retire/RetireForm/index.tsx

```diff
@@ -0,0 +1,460 @@
+    }
+  };
+
+  return (
+    <div>
+      <TwoColLayout>
+        <Col>
+          <div className={styles.offsetCard}>
+            {project ? (
+              <div className={styles.projectHeader}>
+                <ProjectImage category={project.category as CategoryName} />
+                <div className={styles.imageGradient} />
+                <Text t="h3" className={styles.projectHeaderText}>
+                  {project.name || "Error - No project name found"}
+                </Text>
+                <div className={styles.tags}>
+                  <Text className={styles.projectIDText}>
+                    {project.projectID}
+                  </Text>
+                  <Vintage vintage={project.vintageYear} />
+
+                  <Category category={project.category as CategoryName} />
+                  <Registry projectKey={project.registry} />
+                </div>
+              </div>
+            ) : (
+              <div className={styles.projectIDText}>
+                {project && <ProjectImage category="Other" />}
+              </div>
+            )}
+            <div className={styles.offsetCard_ui}>
+              <Text t="caption" color="lighter">
+                <Trans>
+                  Retire carbon credits for yourself, or on behalf of another
+                  person or organization.
+                </Trans>
+              </Text>
+              <Text t="caption" color="lighter">
+                <Trans>
+                  The information below will be broadcast publicly to verify
+                  your environmental claim. This transaction is permanent; the
+                  information cannot be changed once your transaction is
+                  complete.
+                </Trans>
+              </Text>
+              <Text t="body2" color="lightest" className={styles.required}>
+                <Trans>
+                  <span style={{ color: "red" }}>* </span> Required Field
+                </Trans>
+              </Text>
+              <div className={styles.input}>
+                <label>
+                  <div className={styles.stackText}>
+                    <Text t="caption" color="lighter">
+                      <Trans id="offset.amount_in_tonnes">
+                        How many tonnes of carbon would you like to offset?{" "}
+                        <span style={{ color: "red" }}>* </span>
+                      </Trans>
+                    </Text>
+                    <Text
+                      t="body8"
+                      color="lightest"
+                      className={styles.detailsText}
+                    >
+                      <Trans id="offset.amount_in_tonnes_2">
+                        Available: {balance}
+                      </Trans>
+                    </Text>
+                  </div>
+                </label>
+
+                <div className="number_input_container">
+                  <input
```

Comment:
There are already components for `Input` and `TextArea`. 
No need to write them again. 
See [here for example](https://github.com/KlimaDAO/klimadao/blob/staging/carbonmark/components/pages/Project/Purchase/PurchaseInputs.tsx#L59-L94)

Or what is the reason you did not use them?